theme-contract/src/main/java/com/orange/ouds/theme/tokens/semantic/OudsSizeSemanticTokens.kt

```diff
@@ -1,101 +1,101 @@
+  val iconWithBodySmallSizeShortTablet: Float = DimensionRawTokens.dimension150,
+  val iconWithBodySmallSizeTallMobile: Float = DimensionRawTokens.dimension250,
+  val iconWithBodySmallSizeTallTablet: Float = DimensionRawTokens.dimension250,
+  val iconWithHeadingLargeSizeMediumMobile: Float = DimensionRawTokens.dimension550,
+  val iconWithHeadingLargeSizeMediumTablet: Float = DimensionRawTokens.dimension600,
+  val iconWithHeadingLargeSizeShortMobile: Float = DimensionRawTokens.dimension500,
+  val iconWithHeadingLargeSizeShortTablet: Float = DimensionRawTokens.dimension550,
+  val iconWithHeadingLargeSizeTallMobile: Float = DimensionRawTokens.dimension600,
+  val iconWithHeadingLargeSizeTallTablet: Float = DimensionRawTokens.dimension650,
+  val iconWithHeadingMediumSizeMediumMobile: Float = DimensionRawTokens.dimension500,
+  val iconWithHeadingMediumSizeMediumTablet: Float = DimensionRawTokens.dimension550,
+  val iconWithHeadingMediumSizeShortMobile: Float = DimensionRawTokens.dimension400,
+  val iconWithHeadingMediumSizeShortTablet: Float = DimensionRawTokens.dimension500,
+  val iconWithHeadingMediumSizeTallMobile: Float = DimensionRawTokens.dimension550,
+  val iconWithHeadingMediumSizeTallTablet: Float = DimensionRawTokens.dimension600,
+  val iconWithHeadingSmallSizeMediumMobile: Float = DimensionRawTokens.dimension500,
+  val iconWithHeadingSmallSizeMediumTablet: Float = DimensionRawTokens.dimension500,
+  val iconWithHeadingSmallSizeShortMobile: Float = DimensionRawTokens.dimension400,
+  val iconWithHeadingSmallSizeShortTablet: Float = DimensionRawTokens.dimension400,
+  val iconWithHeadingSmallSizeTallMobile: Float = DimensionRawTokens.dimension550,
+  val iconWithHeadingSmallSizeTallTablet: Float = DimensionRawTokens.dimension550,
+  val iconWithHeadingXlargeSizeMediumMobile: Float = DimensionRawTokens.dimension600,
+  val iconWithHeadingXlargeSizeMediumTablet: Float = DimensionRawTokens.dimension700,
+  val iconWithHeadingXlargeSizeShortMobile: Float = DimensionRawTokens.dimension550,
+  val iconWithHeadingXlargeSizeShortTablet: Float = DimensionRawTokens.dimension650,
+  val iconWithHeadingXlargeSizeTallMobile: Float = DimensionRawTokens.dimension650,
+  val iconWithHeadingXlargeSizeTallTablet: Float = DimensionRawTokens.dimension800,
+  val iconWithLabelLargeSizeMedium: Float = DimensionRawTokens.dimension2xs,
+  val iconWithLabelLargeSizeShort: Float = DimensionRawTokens.dimension3xs,
+  val iconWithLabelLargeSizeShorter: Float = DimensionRawTokens.dimension4xs,
+  val iconWithLabelLargeSizeTall: Float = DimensionRawTokens.dimensionXs,
+  val iconWithLabelLargeSizeTaller: Float = DimensionRawTokens.dimensionLg,
+  val iconWithLabelMediumSizeMedium: Float = DimensionRawTokens.dimension4xs,
+  val iconWithLabelMediumSizeShort: Float = DimensionRawTokens.dimension5xs,
+  val iconWithLabelMediumSizeTall: Float = DimensionRawTokens.dimension3xs,
+  val iconWithLabelSmallSizeMedium: Float = DimensionRawTokens.dimension5xs,
```

Comment:
The transitivity should be handled here for all `.dimension4xs` etc. tokens. Closing this PR.